WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostPagerActivity.java

```diff
@@ -218,6 +224,20 @@ public void onBackPressed() {
     }
 
     /*
+     * "bumps" the pageview for the post at the passed position if it hasn't already been done
+     */
+    private void bumpPageViewIfNeeded(int position) {
```

Comment:
Good thinking, but it won't be a problem here. We're only bumping stats in the detail view, which doesn't download newer articles. It does use a ViewPager to swipe between posts, but rather than download new posts it simply enables infinite scroll to get older posts - and these olders posts will have positions that don't already exist.